context/app/flask_static_digest.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+
+from flask import url_for as flask_url_for
+
+
+class FlaskStaticDigest(object):
+    def __init__(self, app=None):
+        self.app = app
+
+        if app is not None:
+            self.init_app(app)
+
+    def init_app(self, app):
+        self.manifest_path = os.path.join(app.static_folder, "public/", "manifest.json")
+        self.has_manifest = os.path.exists(self.manifest_path)
```

Comment:
```suggestion
        self.manifest_path = Path(app.static_folder) / 'public' / 'manifest.json'
        self.has_manifest = self.manifest_path.exists()
```
Prefer using pathlib; References to `manifest_path` would also need to change.